src/kernels/jupyter/types.ts

```diff
@@ -184,6 +184,7 @@ export interface IJupyterServerUriEntry {
     time: number;
     /**
      * An optional display name to show for this server as opposed to just the Uri
+     * @deprecated Used only for migration of display names into the User Provided Server list. Else other providers will have the Display Names.
```

Comment:
`displayName` is deprecated, only used for migration of data from one storage into another